index.html

```diff
@@ -153,6 +154,71 @@ <h1 class="title">Skills</h1>
       </div>
     </div>
   </div>
+  <h1 class="title"> Projects </h1>
+  <div class="projects">
+    <div class="bodycards">
+      <!-- Cotejo Card -->
+      <div class="card">
+        <img class="image" src="./images/projects/CotejoApp.png" alt="CotejoApp">
+        <div class="info">
+          <h2 class="name">CotejoApp</h2>
+          <p class="texts">A funny application to schedule matchs of football.</p>
+        </div>
+        <div class="links">
+          <div class="urls">
+            <a href="https://github.com/Andres2D/cotejo-app-front" target="_blank">
+              <i class="fa-brands fa-github" style="color: black;"></i>
```

Comment:
Use a class instead of a style attribute.

Reply:
I tried but it didn't work